app/src/main/java/org/mozilla/fenix/components/toolbar/DefaultToolbarMenu.kt

```diff
@@ -50,35 +47,53 @@ import org.mozilla.fenix.utils.BrowsersCache
 
 /**
  * Builds the toolbar object used with the 3-dot menu in the browser fragment.
+ * @param context the [Context].
```

Comment:
Nit: Let's add a new line between the description and @params

We have more instances of this type of formatting in our kDocs in the codebase. We should try and move away slowly to be more consistent all around in our codebase.